spec/classes/server_spec.rb

```diff
@@ -47,7 +47,8 @@
             with_content(%r{^storage\.dbPath: /var/lib/mongodb$}).
             with_content(%r{^net\.bindIp:  127\.0\.0\.1$}).
             with_content(%r{^systemLog\.logAppend: true$}).
-            with_content(%r{^systemLog\.path: #{log_path}$})
+            with_content(%r{^systemLog\.path: #{log_path}$}).
+            without_content(%r{^replication\.enableMajorityReadConcern:.*$})
```

Comment:
It's sufficient to use `without_content(%r{replication\.enableMajorityReadConcern:})`

Reply:
Noted. I will update.

Reply:
BTW, thanks for your feedback - this is only my second attempt to contribute to a Puppet/github project, so I'm still learning and any information is appreciated.